PWGCF/FemtoDream/femtoDreamProducerTask.cxx

```diff
@@ -77,14 +75,18 @@ struct femtoDreamProducerTask {
 
   Configurable<bool> ConfDebugOutput{"ConfDebugOutput", true, "Debug output"};
 
+  // Choose if filtering or skimming version is run
+
+  Configurable<bool> ConfIsTrigger{"ConfIsTrigger", true, "Store all collisions"};
+
   /// Event cuts
   FemtoDreamCollisionSelection colCuts;
   Configurable<float> ConfEvtZvtx{"ConfEvtZvtx", 10.f, "Evt sel: Max. z-Vertex (cm)"};
   Configurable<bool> ConfEvtTriggerCheck{"ConfEvtTriggerCheck", false, "Evt sel: check for trigger"};
   Configurable<int> ConfEvtTriggerSel{"ConfEvtTriggerSel", kINT7, "Evt sel: trigger"};
   Configurable<bool> ConfEvtOfflineCheck{"ConfEvtOfflineCheck", false, "Evt sel: check for offline selection"};
 
-  Filter colFilter = nabs(aod::collision::posZ) < ConfEvtZvtx;
+  //Filter colFilter = nabs(aod::collision::posZ) < ConfEvtZvtxSerksnyte;
```

Comment:
What about keeping the filters but adding `!ConfIsTrigger &&` in front?

Reply:
most likely it has to be `ConfIsTrigger == false`...

Reply:
Yes, this is much better. I will implement it.

Reply:
Great. In this case the filters can be kept for derived data production with FemtoDream. I have just checked the latest commits and I think it is not yet done, right? If I overlooked it, let me know and I check again.

Reply:
Please, let me know if the implementation I made at the end is fine for the filter.